app/src/main/res/values-es/about.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="about_title">Acerca De</string>
+    <string name="about_close">Cerrar</string>
+    <string name="about_version">Soy monerujo %1$s (%2$d)</string>
+
+    <string name="donation_text">
+        \"¡Donar bastardos desagradecidos!\"
```

Comment:
¡Donad, bastardos ingratos! *(sounds more like old spanish from spain or pirates)*
¡Donen, bastardos desagradecidos! *(sounds more like generic latin spanish)*

Reply:
I agree with this. The last one would suit better.

Reply:
we want the pirates!

Reply:
![](https://media.giphy.com/media/3o6Zt3KyN0vd1S97d6/giphy.gif)

Reply:
So... do I bring back the pirates?

Reply:
claro que si :)

Reply:
Remember the pirates? They're back, in pog form